src/pages/dashboard/dashboard.hooks.js

```diff
@@ -0,0 +1,26 @@
+import { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { logoutUser } from '../../store/modules/user/actions';
+
+// use testing library hooks
+
+export const useHeader = () => {
+  const [anchorEl, setAnchorEl] = useState(null);
+  const open = Boolean(anchorEl);
+  const dispatch = useDispatch();
+
+  const handleSignOut = () => {
```

Comment:
What do you think to abstract it to another hook like `useAuthorization` so that you can share the auth helpers throughout the whole application?

Reply:
Let's do this in another PR.